app/models/concerns/dateable.rb

```diff
@@ -0,0 +1,28 @@
+# When included in a model, it will create a method for each date field passed in that returns the number of days since
+# the date field was set.
+
+# For example, if you have a model with a field called `submitted_at`, you can call `days_since_submitted` on an
+# instance of that model
+
+# If you want to add a date field to a model, you can do so like this:
+# class MyModel < ApplicationRecord
+#   dateable :submitted_at, :updated_at
+# end
```

Comment:
This is looking good! Thoughts on putting this text into a markdown doc ?

Reply:
To be addressed at a later date.